packages/chai/package.py

```diff
@@ -136,6 +150,8 @@ class Chai(CachedCMakePackage, CudaPackage, ROCmPackage):
         depends_on("umpire+cuda")
         for sm_ in CudaPackage.cuda_arch_values:
             depends_on("umpire+cuda cuda_arch={0}".format(sm_), when="cuda_arch={0}".format(sm_))
+        with when("@2024.02.0:"):
+            depends_on("umpire~fmt_header_only")
```

Comment:
I was under the impression that the header only version works better with CUDA. I might be remembering incorrectly, though.

Reply:
That's the default which was not working, so I was going to try with the library version.